test/sagemaker_tests/pytorch/training/integration/sagemaker/test_neuron.py

```diff
@@ -60,21 +62,21 @@ def invoke_neuron_helper_function(ecr_image, sagemaker_regions, helper_function,
 @pytest.mark.processor("neuron")
 @pytest.mark.model("unknown_model")
 @pytest.mark.neuron_test
-def test_neuron_allreduce_distributed(framework_version, ecr_image, sagemaker_regions, neuron_efa_instance_type):
+def test_neuron_allreduce_distributed(framework_version, ecr_image, sagemaker_regions):
     function_args = {
             'framework_version': framework_version,
-            'instance_type': neuron_efa_instance_type,
+            'instance_type': 'ml.trn1.32xlarge',
```

Comment:
instance type can be declared as a fixture like [this](https://github.com/aws/deep-learning-containers/blob/master/test/sagemaker_tests/pytorch/training/integration/sagemaker/test_smdataparallel.py#L56)